acl/src/main/java/org/apache/rocketmq/acl/common/Permission.java

```diff
@@ -1,13 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package org.apache.rocketmq.acl.common;
 
+import com.alibaba.fastjson.JSONArray;
+import java.util.HashSet;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.rocketmq.acl.plain.PlainAccessResource;
+
 public class Permission {
 
     public static final byte DENY = 1;
     public static final byte ANY = 1 << 1;
     public static final byte PUB = 1 << 2;
     public static final byte SUB = 1 << 3;
 
-    public boolean checkPermission(byte neededPerm, byte ownedPerm) {
+    public static final Set<Integer> ADMIN_CODE = new HashSet<Integer>();
+
+    static {
+        //  UPDATE_AND_CREATE_TOPIC
+        ADMIN_CODE.add(17);
+        //  UPDATE_BROKER_CONFIG
+        ADMIN_CODE.add(25);
+        //  DELETE_TOPIC_IN_BROKER
+        ADMIN_CODE.add(215);
+        // UPDATE_AND_CREATE_SUBSCRIPTIONGROUP
+        ADMIN_CODE.add(200);
+        // DELETE_SUBSCRIPTIONGROUP
+        ADMIN_CODE.add(207);
+    }
```

Comment:
It will be better to use Variable than raw integer.

Reply:
ok，Has been modified